tests/core/test_subdir_data.py

```diff
@@ -137,70 +136,6 @@ def test_get_index_no_platform_with_offline_cache(self, platform=OVERRIDE_PLATFO
 
 
 class StaticFunctionTests(TestCase):
-    def test_read_mod_and_etag_mod_only(self):
```

Comment:
Why is this deleted as part of this PR? 🤔

Reply:
minor deprecation that i forgot to pull out into a separate PR, let me do that